projects/packages/transport-helper/composer.json

```diff
@@ -3,7 +3,9 @@
 	"description": "Package to help transport server communication",
```

Comment:
We need to add the actions.php file to the composer autoload so we run the file when we require the package, like the backups one.